code/script.js

```diff
@@ -230,96 +323,176 @@ const setSecret = () => {
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
+  charactersInPlay = MUSHROOMS
+  board.style.display = 'flex';
+  numberOfGuesses = 0
+  guessSpan.innerHTML = `${numberOfGuesses}`
+  forestAudio.play()
```

Comment:
hm.. the music does not play for me when reloading page/clicking the play sound-button... might be my tired computer. Does it work for you?